src/atoms/IconWithText.js

```diff
@@ -35,21 +35,21 @@ const IconTextRight = styled.div`
 `;
 
 const IconWithText = ({
-	text, iconSize, textSize, circleText, position, fill, textColor, ...rest
+	text, iconSize, textSize, circleText, position, textColor, ...rest
 }) => {
 	switch (position) {
 	case 'right':
 		return (
 			<IconBlockTextRight>
-				<SvgIcon size={iconSize} {...rest} text={circleText || text} fill={fill} textColor={textColor} />
+				<SvgIcon size={iconSize} {...rest} text={circleText || text} textColor={textColor} />
 				<IconTextRight textSize={textSize}>{text}</IconTextRight>
 			</IconBlockTextRight>
 		);
 
 	default:
 		return (
 			<IconBlock>
-				<SvgIcon size={iconSize} {...rest} text={circleText || text} fill={fill} />
+				<SvgIcon size={iconSize} {...rest} text={circleText || text} />
```

Comment:
can we add textColor to this svgIcon also? This way we could remove it as separate prop, and it will be in ...rest

Reply:
I'm not sure.

Reply:
I'm not sure